src/FileFormats/MOF/mof.1.2.schema.json

```diff
@@ -82,6 +82,10 @@
                 "properties": {
                     "name": {
                         "type": "string"
+                    },
+                    "ConstraintDualStart": {
+                        "description": "An initial value for constraint duals that the optimizer may use to warm-start the solution process.",
+                        "type": ["number","null"]
```

Comment:
Okay, so this section actually has to go below in the `oneOf` section, because it needs to be a `number` for scalar-valued constraints, and an `array` for vector-valued constraints.